### ISSUE TITLE
No error when no index is created

### ISSUE THREAD

**Issue by pgulutzan** [tarantool/tarantool#5526]
I have Ubuntu 18.04.
I have Tarantool 2.5, pulled from source today.
I execute these requests:

box.execute([[DROP TABLE add;]])
s=""
  s = "CREATE TABLE add (c0 INT PRIMARY KEY"
for i=1,128 do
  s = s .. ", c" .. i .. " INT UNIQUE"
end
s = s .. ");"
box.execute(s)
box.execute([[SELECT COUNT(*) FROM  "_vindex" WHERE "id" = (SELECT "id" FROM "_vspace" WHERE "name" = 'ADD');]])
box.execute([[INSERT INTO add (c0, c127) VALUES (1, 1);]])
box.execute([[INSERT INTO add (c0, c127) VALUES (2, 1);]])
box.execute([[INSERT INTO add (c0, c128) VALUES (3, 1);]])
box.execute([[INSERT INTO add (c0, c128) VALUES (4, 1);]])

The result of box.execute(s) is: okay.
The result of the SELECT is: 128.
The result of the INSERTs shows: c128 does not have a unique index.
In other words: I create a table with BOX_INDEX_MAX+1 unique columns.
There is no ER_MODIFY_INDEX error but no unique index on c128.
